generation/classifier_free_guidance/2d_ddpm_classifier_free_guidance_tutorial.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<p>Would be better to put [2] on a separate line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>